css/main.css

```diff
@@ -0,0 +1,562 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&family=Inter:wght@400;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+a {
+  text-decoration: none;
+}
+
+hr {
+  width: 80vw;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 15px;
+  background-color: #3c3a39;
+  color: white;
+}
+
+.none {
+  display: none;
+}
+
+#logo {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+.menu-bars {
+  background: transparent;
+  border: none;
+  outline: none;
+}
+
+.menu-bars i {
+  width: 18px;
+  height: 12px;
+  color: white;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+}
+
+#intro,
+#about-me {
+  background-color: #1c1a19;
+  color: white;
+  position: relative;
+}
+
+#intro {
+  padding: 10%;
+  padding-bottom: 20vh;
+}
+
+.intro-icon {
+  position: absolute;
+  right: 0;
+  bottom: 0;
+}
+
+.desktop-icon {
+  display: none;
+}
+
+.card h2 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.intro-greet,
+#about-me h2 {
+  color: #ff6b00;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+}
+
+.intro-me {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 32px;
+  align-items: center;
+  color: #fff5e1;
+}
+
+.intro-text {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+.social {
+  display: flex;
+}
+
+.social-link {
+  padding: 5px;
+  background-color: transparent;
+  transition: 350ms;
+}
+
+.social-link:hover {
+  background-color: rgba(255, 255, 255, 0.096);
+}
+
+.social-icon {
+  height: 20px;
+}
+
+.about-me-section,
+.stacks {
+  position: relative;
+  padding: 10%;
+  padding-bottom: 20vh;
+}
+
+#projects {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  background-color: #e5e5e5;
+  padding: 10%;
+}
+
+#projects h2 {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #1c1a19;
+}
+
+#projects hr {
+  margin: 25px 0;
+  border-color: #212121;
+}
+
```

Comment:
- [ ] REQUIRED: Prefer using class selector instead of id. both are used to target elements in CSS but the difference is that `CSS class` applies the style to multiple elements(which makes it reusable)  while `ID` applies the style to only one unique element.